src/Components/SmartComponents/Modals/__snapshots__/ReportConfigModal.test.js.snap

```diff
@@ -193,170 +193,6 @@ exports[`Report config modal component Should match snapshots 1`] = `
                                 type="text"
                                 value=""
                               />
-                              <div
-                                class="pf-c-check pf-u-mt-md"
-                              >
-                                <input
-                                  aria-invalid="false"
-                                  class="pf-c-check__input"
-                                  id="hasRule"
-                                  type="checkbox"
-                                />
```

Comment:
I don't know where but all the checkbox and radio glyphs got lost :smile: 
![checkbox](https://user-images.githubusercontent.com/8426204/106162373-50901b00-6188-11eb-8632-5fdf2c32cbed.png)
![radio](https://user-images.githubusercontent.com/8426204/106162379-51c14800-6188-11eb-8fef-229e2449a675.png)
![params](https://user-images.githubusercontent.com/8426204/106162382-538b0b80-6188-11eb-988c-c1548d69a362.png)

Reply:
What browser do you use?

Reply:
Chrome

Reply:
hm, interesting same here. Need to check PF version

Reply:
@leSamo I fixed it. The issue was the width I set for `input, textarea`.I use a class to specifically set the width for the title input and the notes textarea.  I bumped also PF version.